src/Command/MoveModulesToComposerCommand.php

```diff
@@ -220,15 +220,19 @@ private function getComposerPackageName(string $moduleName): string
 
     private function clearDrupalCache(): void
     {
-        $command = [
-            'drush',
-            'cache:rebuild',
-        ];
+        $drushCommand = 'drush';
         if ($this->siteUri) {
-            $command[] = sprintf('--uri=%s', $this->siteUri);
+            $drushCommand .= sprintf(' --uri=%s', $this->siteUri);
         }
 
-        $process = new Process($command, $this->projectPath);
+        $command = sprintf(
+            "echo \"SHOW TABLES LIKE 'cache%%'\" | $(%s sql-connect) | tail -n +2 | xargs -L1 -I%% echo \"TRUNCATE TABLE %%;\" | $(%s sql-connect) -v && %s cr",
```

Comment:
Not an easy one to understand.

`tail -n +2` is for the removal of the warning and header?

Is there a reason to use `sql-connect` over `sql-query`? 
It looks like it could be simpler and more stable with `sql-query`.

Reply:
Well I admit I found the code on the internet and I didn't really try to understand what it did

I will try your hints.

Reply:
I was able to simplify the command a bit